CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+
+1.1.0-M1 / 2020-12-29
+==================
```

Comment:
Shouldn't we mention the support of inline classes and performance optimizations in the plugin?

Reply:
Welp, this was for 1.4.30-M1 that didn't have necessary commits. I'll change all versions from M1 to RC, add inline classes and reopen